src/alchemlyb/parsing/amber.py

```diff
@@ -0,0 +1,265 @@
+"""Parsers for extracting alchemical data from amber output files.
+Most of the file parsing part are inheriting from alchemical-analysis  
+Change the final format to pandas to be consistent with the alchemlyb format
+"""
+
+import pandas as pd
+import re
+import numpy as np
+import os
+
```

Comment:
Use [logging](https://docs.python.org/3/library/logging.html) instead of `print`:
```
import logging

logger = logging.getLogger("alchemlyb.parsers.Amber")
```

See more comments below.

Note that in order to see the messages, a root logger has to be created elsewhere:
```python
import logging

def create_alchemlyb_logger(logfile="alchemlyb.log", name="alchemlyb"):
    """Create a logger that outputs to screen and to `logfile`"""

    logger = logging.getLogger(name)
    logger.setLevel(logging.DEBUG)

    # handler that writes to logfile
    logfile_handler = logging.FileHandler(logfile)
    logfile_formatter = logging.Formatter('%(asctime)s %(name)-12s %(levelname)-8s %(message)s')
    logfile_handler.setFormatter(logfile_formatter)
    logger.addHandler(logfile_handler)

    # define a Handler which writes INFO messages or higher to the sys.stderr
    console_handler = logging.StreamHandler()
    console_handler.setLevel(logging.INFO)
    # set a format which is simpler for console use
    formatter = logging.Formatter('%(name)-12s: %(levelname)-8s %(message)s')
    console_handler.setFormatter(formatter)
    logger.addHandler(console_handler)

    return logger

rootlogger = create_alchemlyb_logger()
```
We might add this convenience function elsewhere in the library. I opened issue #34. 

Note that you only have to create the top level logger (here called `rootlogger`) once. Loggers are known globally in the interpreter, so you can "attach" to the root logger from anywhere else just by naming the new logger "alchemlyb.OTHER.STUFF".

Reply:
Yes, will switch to logging instead of printing. 

Just want to make sure that I understand the logic correctly, so what I need to do is to add 

```
import logging

logger = logging.getLogger("alchemlyb.parsers.Amber")
```

at the beginning of the code and start to logging like

logging.info ("Write some logging info here") is that correct?

Reply:
Yes, exactly.

(Just be aware that until a logger named "alchemlyb" has been created, no logging will be visible, but that's expected and desired behavior. With #34 I will add code to the library to start logging automatically.)